### PR TITLE
DCAC-25 - Create skeleton application for item-group-workflow-api

### PR DESCRIPTION
**Pull request by caismith-ch**
Add relevant files for skeleton application.
Resolves: DCAC-25

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+# Actuator health check config
+management.endpoints.web.base-path=/
+management.endpoints.web.path-mapping.health=healthcheck
+
+# Default application root path
+server.servlet.context-path=/item-group-workflow-api
+
+server.port=9090
```
Comment (hwilliams1) [inline src/main/resources/application.properties:9]: Shouldn't be including the server.port when committing the code to the repository

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    <version>unversioned</version>
+    <name>item-group-workflow-api</name>
+    <description>Rest API to handle item group workflow requests.</description>
+    <packaging>jar</packaging>
+
+    <properties>
+        <java.version>11</java.version>
+        <spring-boot-dependencies.version>2.7.8</spring-boot-dependencies.version>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
```
Comment (hwilliams1) [inline pom.xml:23]: Are these still needed with the spring-boot-maven-plugin?

**Review by hwilliams1 — CHANGES_REQUESTED** [review]
Should remove the server.port and see if the maven.compiler.source is actually needed or not

File: src/main/resources/application.properties (hunk 1 above)
Comment (caismith-ch) [reply src/main/resources/application.properties:9]: good spot, will remove

File: pom.xml (hunk 2 above)
Comment (caismith-ch) [reply pom.xml:23]: will test and let you know

**Review by hwilliams1 — APPROVED** [review]

**Pull request merged**